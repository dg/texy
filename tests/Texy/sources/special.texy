[* obrazek.jpg .(alternativni text){float: left; title: atribut title; lang: en} *]

10 % a 20 %

m^2 a m_2 a r_1^3

  {{test:test}}
{{aa}} {{bbb}}


>>>>>

> test1
>> test2
>>>> test3
> test4
>>
>>>>>
>

> test
>>>> test
> test

*my+ **strong*** konec** jo?*

*a*

**nested .(** asterix)**

escape \**asterix**

"hello .(test (vnorene\) zavorky)"

{{ script }}

texyscript {{ texy }}
texyscript2 {{ texy (test) }}

obrazek [* obr.jpg *]:[* popup.php?img=obr-big.jpg *]

[* obr.jpg .{alt:alternative; title:title text }*]:[* popup.php?img=obr-big.jpg *] *** Nějaký popisek pod obrázkem

absolutní obr [* http://www.texy.info/images/buttons/texy-powered.png *] test

[* 0 *] nebo **0**

`slovo:[url]` a sl0ovo:[url] a slovo_:[url] a sl-aovo:[url]

Neexistujici &entita;

Specialní &amp;image;

a --> b <--> c <-- d ===> e



Šlitr-Suchý Šlitr--Suchý

123-456

456--456

456 --456

7869-- 4646

Šlitr-- Suchý

Šlitr --Suchý

Šlitr -- Suchý

Šlitr - Suchý


321-654-987

- (phone) number 1 123 123 123
- (phone) number 1 123 123
- number 1 123

**email@email.cz**

**www.example.com**

"www.example.com/adresa/dlouha/moc.php":www.example.cz

m^-2

m^2+

m^2n

m_2n


- http://cs.wikipedia.org/wiki/Prechodnik
- http://cs.wiki­pedia.org/wiki/Přechod­ník

10+-3
+-25


[**]

[* *]

[* <>]

[* image <>]

[* ><]

[* >]

[* <]

.<>
[* image *]


//V zahradě rozkoší zachmuřen
 "Hieronymus Bosch":[http://en.wikipedia.org/wiki/Hieronymus_Bosch]
 v přístěnku Bestiář Aristotelův//


200 Kč | 3 990 Kč | 5 000 000 Kč


Hokus  -- pokus

- ftp://example.com
- ftps://example.com
- ftps://user:pass@example.com

'This guy's really nice.'



1. test


1. test
2. test



2. test



1.test



1. test
2.test


<h1>"neco":</h1>
