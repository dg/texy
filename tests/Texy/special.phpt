<?php

/**
 * Test: Special cases.
 */

use Tester\Assert;

require __DIR__ . '/../bootstrap.php';


$texy = new Texy\Texy;
$texy->linkModule->root = 'xxx/';
$texy->imageModule->root = '../images/';
$texy->imageModule->linkedRoot = '../images/big/';
$texy->imageModule->leftClass = 'left';
$texy->htmlOutputModule->lineWrap = 180;
$texy->allowed['longwords'] = FALSE;
$texy->typographyModule->locale = 'en';

Assert::matchFile(
	__DIR__ . '/expected/special.html',
	$texy->process(file_get_contents(__DIR__ . '/sources/special.texy'))
);
