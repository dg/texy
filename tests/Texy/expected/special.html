<div class="figure"><img title="atribut title" lang="en" style="float:left" src="../images/obrazek.jpg" alt="alternativni text" /></div>

<p>10 % a 20 %</p>

<p>m<sup>2</sup> a m<sub>2</sub> a r<sub>1</sub><sup>3</sup></p>

<p>{{test:test}} {{aa}} {{bbb}}</p>

<p>&gt;&gt;&gt;&gt;&gt;</p>

<blockquote>
	<p>test1</p>
</blockquote>

<p>&gt;&gt; test2 &gt;&gt;&gt;&gt; test3</p>

<blockquote>
	<p>test4</p>
</blockquote>

<p>&gt;&gt; &gt;&gt;&gt;&gt;&gt; &gt;</p>

<blockquote>
	<p>test</p>
</blockquote>

<p>&gt;&gt;&gt;&gt; test</p>

<blockquote>
	<p>test</p>
</blockquote>

<p><em>my+ <strong>strong*** konec</strong> jo?</em></p>

<p><em>a</em></p>

<p><strong title="** asterix">nested</strong></p>

<p>escape **asterix**</p>

<p><span title="test (vnorene) zavorky">hello</span></p>

<p>{{ script }}</p>

<p>texyscript {{ texy }} texyscript2</p>

<p>obrazek <a href="../images/big/popup.php?img=obr-big.jpg" onclick="return !popupImage(this.href)"><img src="../images/obr.jpg" alt="" /></a></p>

<div class="figure"><a href="../images/big/popup.php?img=obr-big.jpg" onclick="return !popupImage(this.href)"><img alt="alternative" title="title text" src="../images/obr.jpg"
/></a>
	<p>Nějaký popisek pod obrázkem</p>
</div>

<p>absolutní obr <img src="http://www.texy.info/images/buttons/texy-powered.png" alt="" /> test</p>

<p><img src="../images/0" alt="" /> nebo <strong>0</strong></p>

<p><code>slovo:[url]</code> a <a href="xxx/url">sl0ovo</a> a <a href="xxx/url">slovo_</a> a <a href="xxx/url">sl-aovo</a></p>

<p>Neexistujici &amp;entita;</p>

<p>Specialní &amp;image;</p>

<p>a → b ↔ c ← d ⇒ e</p>

<p>Šlitr-Suchý Šlitr–Suchý</p>

<p>123–456</p>

<p>456–456</p>

<p>456 –456</p>

<p>7869– 4646</p>

<p>Šlitr– Suchý</p>

<p>Šlitr –Suchý</p>

<p>Šlitr – Suchý</p>

<p>Šlitr – Suchý</p>

<p>321–654–987</p>

<ul>
	<li>(phone) number 1 123 123 123</li>

	<li>(phone) number 1 123 123</li>

	<li>number 1 123</li>
</ul>

<p><strong><a href="mailto:email&#64;email.cz">email&#64;<!-- -->email.cz</a></strong></p>

<p><strong><a href="http://www.example.com">www.example.com</a></strong></p>

<p><a href="http://www.example.cz">www.example.com/…ouha/moc.php</a></p>

<p>m<sup>−2</sup></p>

<p>m<sup>2+</sup></p>

<p>m<sup>2n</sup></p>

<p>m<sub>2n</sub></p>

<ul>
	<li><a href="http://cs.wikipedia.org/wiki/Prechodnik">http://cs.wikipedia.org/wiki/Prechodnik</a></li>

	<li><a href="http://cs.wikipedia.org/wiki/Přechodník">http://cs.wikipedia.org/wiki/Přechodník</a></li>
</ul>

<p>10±3 ±25</p>

<p>[**]</p>

<p>[* *]</p>

<p>[* &lt;&gt;]</p>

<p>[* image &lt;&gt;]</p>

<div style="float:left" class="figure"><img src="../images/&gt;" alt="" /></div>

<p>[* &gt;]</p>

<p>[* &lt;]</p>

<div style="text-align:center"><img src="../images/image" alt="" /></div>

<p><em>V zahradě rozkoší zachmuřen<br />
<a href="http://en.wikipedia.org/wiki/Hieronymus_Bosch">Hieronymus Bosch</a><br />
v přístěnku Bestiář Aristotelův</em></p>

<p>200 Kč | 3 990 Kč | 5 000 000 Kč</p>

<p>Hokus – pokus</p>

<ul>
	<li><a href="ftp://example.com">ftp://example.com</a></li>

	<li>ftps://example.com</li>

	<li>ftps://user:pass@example.com</li>
</ul>

<p>‘This guy's really nice.’</p>

<p>1. test</p>

<ol>
	<li>test</li>

	<li>test</li>
</ol>

<p>2. test</p>

<p>1.test</p>

<p>1. test 2.test</p>

<h1>“neco”:</h1>
